src/mgr/BaseMgrStandbyModule.cc

```diff
@@ -128,6 +128,23 @@ ceph_log(BaseMgrStandbyModule *self, PyObject *args)
   Py_RETURN_NONE;
 }
 
+static PyObject*
+ceph_cluster_log(BaseMgrStandbyModule *self, PyObject *args)
```

Comment:
Can remove these from the standby module definition now that we're not exposing the functionality for standby modules